org.civicrm.reqangular/test/mocks/services/api/option-group-mock.js

```diff
@@ -129,20 +129,44 @@ define([
               id: '2',
               option_group_id: '22',
               label: 'Region #2',
-              value: 'Region #2',
+              value: '1',
```

Comment:
why did you change this value and not the others? should this have been changed at all?

Reply:
> should this have been changed at all?

This value should be consistent, numeric string.

![image](https://user-images.githubusercontent.com/3973243/31438549-611b7ce4-ae80-11e7-9f64-687c1df075da.png)

Updated mocks.